examples/adv_diff/ex8/BoussinesqForcing.cpp

```diff
@@ -0,0 +1,203 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2014 - 2019 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+#include "BoussinesqForcing.h"
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include <SAMRAI_config.h>
```

Comment:
```suggestion
```
Not super important but I'm trying to get rid of all of these. SAMRAI headers always include this header.